jupyter-js-widgets/src/widget_selection.js

```diff
@@ -187,10 +284,21 @@ var DropdownView = widget.DOMWidgetView.extend({
         this.dropbutton.blur();
 
         if (this.droplist.classList.contains('mod-active')) {
+            // Deselect active item.
+            var active = this.droplist.querySelector('.mod-active');
+            if (active) {
+                active.classList.remove('mod-active');
+            }
+            // Close the drop list.
             this.droplist.classList.remove('mod-active');
+            // Remove global event listener.
+            document.removeEventListener('keydown', this.keydown);
             return;
         }
 
+        // Add a global keydown listener for drop list events.
+        document.addEventListener('keydown', this.keydown);
```

Comment:
FYI, here's Bootstrap attached a `document` listener that it then filters with its `toggle` selector:

https://github.com/twbs/bootstrap/blob/master/js/dropdown.js#L162